files/check_zpool_scrub

```diff
@@ -0,0 +1,527 @@
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+# CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+########################################################################
+# Date functions
+########################################################################
+
+# This date function must be placed on the top of this file because
+# they are used in some global variables.
+
+# to_year ###
+
+##
+# Get the four digit year integer from now.
+#
+# Return:
+#   The current 4 digit year.
+##
+now_to_year() {
+	date +%Y
+}
+
+##
+# Convert a date in the format YYYY-MM-DD to a four digit year integer.
+#
+# Parameters:
+#   a date in the format YYYY-MM-DD
+#
+# Return:
+#   four digit year integer
+##
+date_to_year() {
+	date --date $1 +%Y
+}
+
+# to_datetime ###
+
+##
+# Convert a UNIX timestamp to a datetime string.
+#
+# Parameters:
+#   UNIX timestamp
+#
+# Return:
+#   %Y-%m-%d.%H:%M:%S
+##
+timestamp_to_datetime() {
+	date --date @$1 +%Y-%m-%d.%H:%M:%S
+}
+
+# to_timestamp ###
+
+##
+# Get the current UNIX timestamp.
+#
+# Return:
+#   %current UNIX timestamp
+##
+now_to_timestamp() {
+	date +%s
+}
+
+##
+# Convert a datetime in the ctime format to a Unix timestamp.
+# ctime is a textual representation of a datetime derived from the
+# c function ctime (https://en.cppreference.com/w/c/chrono/ctime).
+# The ctime string has the following format:  Www Mmm dd hh:mm:ss yyyy,
+# where Www is the weekday, Mmm the month in letters, dd the day of the
+# month, hh:mm:ss the time, and yyyy the year.
+#
+# see https://www.freebsd.org/cgi/man.cgi?query=strftime&sektion=3
+#  %c is replaced by national representation of time and date.
+#
+# Parameters:
+#   $1: Www Mmm dd hh:mm:ss yyyy
+#
+# Return:
+#   UNIX timestamp
+##
+ctime_to_timestamp() {
+	date --date "$1" +%s
+}
+
+########################################################################
+# Global variables
+########################################################################
+
+. "$(dirname "${0}")/common.sh"
+
+checkName=zpool-scrub
+checkOutput=""
+
+USAGE="
+Usage: $0 <options>
+
+Options:
+ -c, --critical=OPT_CRITICAL
+    Interval in seconds for critical state.
+ -w, --warning=OPT_WARNING
+    Interval in seconds for warning state. Must be lower than -c.
+ -p,--pool=OPT_POOL
+    Name of the pool. If this option is omitted all pools are checked.
+ -h, --help
+    Show this help.
+
+Performance data:
+
+POOL is the name of the pool
+
+ - POOL_last_ago
```

Comment:
Why do you sometimes use underscores and sometimes you don't?

Reply:
I didn't change that part. I will make it consistent now.